build/build.sh

```diff
@@ -71,9 +71,12 @@ else
 	)
 fi
 
-# transform TAGS to build args
 IMAGE_NAME=$(jq -r '.name' package.json)
```

Comment:
Could we also just change the name in `package.json`? Without any code change?

Reply:
No, I think this is not a good idea. I think we need to keep the -lts around until we finally publish to dockerhub only. Otherwise we need to change too many things, e.g. local image versioning vs latest only, how to identify images when pushed to/pulled from GHCR...